config.md

```diff
@@ -26,18 +26,22 @@ For all platform-specific configuration values, the scope defined below in the [
 
 ## <a name="configRoot" />Root
 
-**`root`** (object, REQUIRED) specifies the container's root filesystem.
+**`root`** (object, OPTIONAL) specifies the container's root filesystem.
+    On Windows, for Windows Server Containers, this field is REQUIRED.
+    For [Hyper-V Containers](config-windows.md#hyperv), this field MUST NOT be set.
 
-* **`path`** (string, OPTIONAL) Specifies the path to the root filesystem for the container.
-    The path is either an absolute path or a relative path to the bundle.
+    On all other platforms, this field is REQUIRED.
 
-    * On Windows, for Windows Server Containers, this field is REQUIRED and MUST be specified as a [volume GUID path][naming-a-volume].
-      For Hyper-V Containers, this field MUST be omitted.
-    * On all other platforms, this field is REQUIRED.
+* **`path`** (string, REQUIRED) Specifies the path to the root filesystem for the container.
+
+    * On Windows, `path` MUST be a [volume GUID path][naming-a-volume].
+
+    * On POSIX, `path` is either an absolute path or a relative path to the bundle.
```

Comment:
> "On POSIX" doesn't make any sense -- POSIX isn't a platform, it's a specification, and it's one that Windows is compliant with in many cases.

I've changed “On POSIX” to “On POSIX platforms” in 589d2eb → 767eeaf.

> I'd perfer to see this say either "On Unix" or "On Unix-like platforms".

What is the gap between “Unix-like platforms” and “POSIX platforms”?  “Unix-like platforms” seems too vague for a spec, and if we want a name for non-Windows that is distinct from POSIX, I think we need to define that name [when we define our compliance tracks][1].  I'd suggested that on the 1st, but @crosbymichael [prefered to punt on that for now][2].

Looking at the properties I'm associating with POSIX here:

* For `root.path`, POSIX platforms clearly do not include Windows, because [POSIX paths are slash-separated][3].
* `mounts[].type` is less clearly POSIX-specific (or even Unix-like-specific), since POSIX defines neither `mount(8)` nor `mount(2)`.  However, it's also unclear to me why Windows only supports volume GUID paths.  Is that a limitation of the Windows platform, or just a limitation of the current Windows runtime?
* `process.user.uid` and friends are POSIX-platform-specific, since numeric [user][4] and [group][5] IDs are defined by POSIX.  I'm not aware of numeric user/group IDs on Windows, but if it does support them, I'd expect Windows runtimes to support `process.user.uid`, etc. as well.
* `hooks` is less clearly POSIX-specific (or even Unix-like-specific), since these seem easy to implement regardless of your platform.  I suspect this is a limitation of the current Windows runtime, and not a limitation of the Windows platform, so tying it to POSIX platforms seems no worse than tying it to Unix-like platforms.  Maybe we want to leave this one as an explicit “Linux and Solaris Hooks” or “Hooks (not Windows)” or some such?

[1]: https://github.com/opencontainers/runtime-spec/blob/fd0b0b6f7505ba8d24ee1fe59e59ca969b26843a/spec.md#platforms
[2]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2017/opencontainers.2017-06-01-17.41.log.html#l-60
[3]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13
[4]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_436
[5]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_189